### PR TITLE
Add describe tasks-config command

### PR DESCRIPTION
**Pull request by mimaison**
This maps to /connectors/<connector>/tasks-config and returns the configuration of all tasks of a connector.
Usage: kcctl describe tasks-config <connector>

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Hey @mimaison, welcome, and thanks a lot for this contribution! I think it's a great addition, two questions only on the way of delivering this feature:

* This end-point is only available in AK 2.8 and newer, invoking the command against an earlier version currently prints an ugly stack trace due to the 404 returned by the REST API. Should we query the version (it already is exposed via `kcctl info`)  and display a nice error message in case the API doesn't exist?
* Should this be its own sub-command under `describe` indeed, or should we rather show the tasks config upon `describe connector <connector id>`? Thinking about it, this would allow for a transparent handling of the first issue, as we simply would or would not show the tasks config based on the backend's capabilities. Note that `describe` already yields an aggregated view, i.e. there's no 1:1 relationship between API methods and kcctl commands. Or do you think this would be too much info to show for a single command?

**Comment by gunnarmorling** [comment]
Hey @mimaison, howdy? Any thoughts on the questions above?

**Comment by mimaison** [comment]
Hi @gunnarmorling

Thanks for the feedback, I'll hopefully have the time to update the PR tomorrow.

- You're right, this endpoint is still very new. Many users probably don't run 2.8 yet so it makes sense to check compatibility and handle older versions gracefully.
- I wondered of the best way to expose this feature. I think task-configs is a lot of details not all users may want to see all the time. Also with many tasks it can be very verbose! So I wasn't sure if adding it to `describe` was the best option, that's why I added a new command. Another option is to add an extra flag to `describe` to get all the details. WDYT?

**Comment by gunnarmorling** [comment]
Good point re verbosity. In that light, yes, adding a flag to `describe` would be my preference, I think. It feels more consistent in a way.

**Comment by mimaison** [comment]
Thanks @gunnarmorling 
I've pushed an update. Let me know if you have any feedback.

File: src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java [hunk 1]
```diff
@@ -85,17 +93,37 @@ public void run() {
...
-        printTuples(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
+        Tuple.print(Arrays.asList(new Tuple(ANSI_WHITE_BOLD + "Topics" + ANSI_RESET, "")));
 
         List<Tuple> topics = new ArrayList<>();
 
         for (String topic : connectorTopics.entrySet().iterator().next().getValue().topics) {
             topics.add(new Tuple("", "  " + topic));
         }
-        printTuples(topics);
+        Tuple.print(topics);
```
Comment (gunnarmorling) [inline src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java:106]: Let's put the tasks config above topics. That way the configs (connector + tasks) is next to each other which I feel is more consistent.

File: src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java [hunk 2]
```diff
@@ -85,17 +93,37 @@ public void run() {
...
+                for (Entry<String, Map<String, String>> task : tasksConfigs.entrySet()) {
+                    tuples.add(new Tuple("  Task", task.getKey()));
+                    for (Entry<String, String> taskConfig : task.getValue().entrySet()) {
+                        tuples.add(new Tuple("    " + taskConfig.getKey(), taskConfig.getValue()));
+                    }
+                }
+                Tuple.print(tuples);
+            }
+            else {
+                System.out.println("  --tasks-config requires at least Kafka Connect 2.8. Current version: " + currentVersion);
```
Comment (gunnarmorling) [inline src/main/java/dev/morling/kccli/command/DescribeConnectorCommand.java:124]: So do you think we actually should show the other output in this situation; or should we rather handle this as sort of an invocation failure more explicitly and _only_ show this message when passing the parameter with KC < 2.8?

**Review by gunnarmorling — APPROVED** [review]
This looks mostly good now, @mimaison! One more suggestion and one question inline. Besides that, could you also

* Log an issue for this feature
* Reword the commit message so it begins with the issue key and an emoji of your liking: "#123 🔎 Adding support for ..."; there shall be no emoji-less commit in this repo ;)

Thanks again!

**Comment by mimaison** [comment]
Thanks @gunnarmorling!
I've addressed your suggestions.

**Comment by gunnarmorling** [comment]
Excellent, applied. Thank you so much, @mimaison! New binaries with this change will be available automatially in a few min.

**Comment by gunnarmorling** [comment]
Pushed one more commit for updating the completion script. That's a bit tedious atm., looking into making this process a bit smoother currently.

**Pull request merged**